cli/src/args/topic.rs

```diff
@@ -78,10 +79,18 @@ pub(crate) struct TopicCreateArgs {
     pub(crate) partitions_count: u32,
     /// Name of the topic
     pub(crate) name: String,
+    /// Max topic size
+    /// ("unlimited" or skipping parameter disables max topic size functionality in topic)
```

Comment:
Please add empty line between those lines - after that first line will be printed as a help when `-h` is used and all first line and remaining part will be shown when using `--help`